modules/assimp/editor_scene_importer_assimp.cpp

```diff
@@ -150,8 +150,7 @@ Node *EditorSceneImporterAssimp::import_scene(const String &p_path, uint32_t p_f
 								 0;
 	const aiScene *scene = importer.ReadFile(s_path.c_str(),
 			post_process_Steps);
-	ERR_EXPLAIN(String("Open Asset Import failed to open: ") + String(importer.GetErrorString()));
-	ERR_FAIL_COND_V(scene == NULL, NULL);
+	ERR_FAIL_COND_V_MSG(scene == NULL, NULL, String("Open Asset Import failed to open: ") + String(importer.GetErrorString()) + ".");
```

Comment:
Could be `ERR_FAIL_NULL_V_MSG(scene, ...)`, though it's also fine to keep as is in this PR. A regex could likely fix current occurrences of `ERR_FAIL_COND.*([A-Za-z0-9_]* == NULL`.

Reply:
Keeping as it is for now at least